docs/website/content/v0.3/en/components/apid.md

```diff
@@ -0,0 +1,47 @@
+---
+title: 'apid'
+---
+
+When interacting with Talos, the gRPC api endpoint you will interact with directly is APId.
+APId acts as the gateway for all component interactions.
+APId provides a mechanism to route requests to the appropriate destination.
```

Comment:
In this paragraph the service is referred to as "APId", and in the next it is "apid". Not sure I care too much, but these should probably align. Adding that a bit, in other component docs we use the reformatted lowercase version (e.g.`apid`) of the name. To give a consistent feel, we should do that in this doc too.